octoprint_mrbeam/static/js/material_settings.js

```diff
@@ -43,16 +46,18 @@ $(function () {
                     }
                 })
                 .fail(function (response) {
-                    setTimeout(
-                        self.loadMaterialSettings,
-                        self.MATERIAL_SETTINGS_RETRY_TIME,
-                        callback
-                    );
-                    console.error(
-                        "Unable to load material settings. Retrying in " +
+                    if (self.loginState.loggedIn()) {
```

Comment:
Please add an else statement and log a message like "Unable to load material settings. Not retrying because user is not logged in."